gcloud-java-examples/README.md

```diff
@@ -89,6 +89,20 @@ To run examples from your command line:
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="download <bucket_name> test.txt"
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="delete <bucket_name> test.txt"
     ```
+ * Here's an example run of `Stat`.
```

Comment:
I'm +1 for adding an example that provides extra guidance in a separate PR.  It's nice to have more examples that provide guidance on usage.